tests/test_query.py

```diff
@@ -6,6 +6,30 @@
 from fastapi import status
 
 
+@pytest.fixture()
+def mock_token():
+    """Create a mock token that is well-formed for testing purposes."""
+    return "Bearer foo"
+
+
+@pytest.fixture()
+def mock_verify_token():
+    """Mock a successful token verification that does not raise any exceptions."""
+
+    def _verify_token(token):
+        return None
+
+    return _verify_token
+
+
+@pytest.fixture()
+def set_mock_verify_token(monkeypatch, mock_verify_token):
+    """Set the verify_token function to a mock that does not raise any exceptions."""
+    monkeypatch.setattr(
+        "app.api.routers.query.verify_token", mock_verify_token
+    )
+
+
```

Comment:
That's pretty neat. I am not sure I've seen that before. When I use this fixture in a test, I immediately use this monkeypatch?

Reply:
guess we could use that for our mocked `get` methods in the future?

Reply:
> When I use this fixture in a test, I immediately use this monkeypatch?

that's right :)

we actually also are using this trick in the API, to set some toy API credentials for the tests: https://github.com/neurobagel/api/blob/7f03522a9dff74c1f89d60cd398b2204457a65f4/tests/conftest.py#L14-L18